auslib/db.py

```diff
@@ -1899,7 +1920,7 @@ def insert(self, changed_by, transaction=None, dryrun=False, signoffs=None, **co
             raise PermissionDeniedError("%s is not allowed to create releases for product %s" % (changed_by, columns["product"]))
 
         if not dryrun:
-            potential_required_signoffs = self.getPotentialRequiredSignoffs([columns], transaction=transaction)
+            potential_required_signoffs = next(iter(self.getPotentialRequiredSignoffs([columns], transaction=transaction).values()))
```

Comment:
This class' `getPotentialRequiredSignoffs` may return a dict with multiple values, which means you may miss some of the required signoffs. You should iterate over all of the returned key/value pairs to make sure all of them are taken into account. It would be best to do this for all of the other places where getPotentialRequiredSignoffs is called outside of a loop or comprehension, too.

Reply:
Hadn't thought of this. Let me fix this ASAP. Thanks